docs/package.json

```diff
@@ -5,6 +5,98 @@
   "author": "Material-UI Team",
   "license": "MIT",
   "scripts": {
-    "typescript": "tslint -p tsconfig.json \"src/pages/**/*.{ts,tsx}\""
+    "build": "rimraf .next && cross-env NODE_ENV=production next build",
+    "build-sw": "node ./scripts/buildServiceWorker.js",
+    "dev": "rimraf ../node_modules/.cache/babel-loader && cross-env NODE_ENV=development node src/server.js",
+    "deploy": "git push material-ui-docs master:latest",
+    "export": "rimraf docs/export && next export -o export && yarn build-sw && cp -r netlify/. export",
+    "icons": "rimraf static/icons/* && node ./scripts/buildIcons.js",
+    "start": "next start",
+    "typescript": "tslint -p tsconfig.json \"src/pages/**/*.{ts,tsx}\"",
+    "typescript:transpile": "node scripts/formattedTSDemos",
+    "typescript:transpile:dev": "node scripts/formattedTSDemos --watch"
+  },
+  "dependencies": {
+    "@date-io/date-fns": "^1.0.0",
+    "@emotion/core": "^10.0.0",
+    "@emotion/styled": "^10.0.0",
+    "@material-ui/core": "^4.1.2",
+    "@material-ui/docs": "^4.0.0-beta.0",
+    "@material-ui/icons": "^4.2.1",
+    "@material-ui/lab": "^4.0.0-alpha.18",
+    "@material-ui/pickers": "^3.1.1",
+    "@material-ui/styles": "^4.1.2",
+    "@material-ui/system": "^4.3.0",
+    "@trendmicro/react-interpolate": "^0.5.5",
+    "@types/autosuggest-highlight": "^3.1.0",
+    "@types/react-autosuggest": "^9.3.7",
+    "@types/react-dom": "^16.0.9",
+    "@types/react-router-dom": "^4.3.1",
+    "@types/react-select": "^2.0.14",
+    "@types/react-swipeable-views": "^0.13.0",
+    "@types/react-text-mask": "^5.4.2",
+    "@types/react-virtualized": "^9.21.1",
+    "@types/react-window": "^1.7.0",
+    "@types/styled-components": "4.1.12",
+    "@zeit/next-typescript": "^1.1.1",
+    "address": "^1.0.3",
+    "ast-types": "^0.12.4",
+    "autoprefixer": "^9.0.0",
+    "autosuggest-highlight": "^3.1.1",
+    "classnames": "^2.2.6",
```

Comment:
A quick search in VSCode shows this isn't used anymore

Reply:
This is a peer dependency of one of our dependencies as far as I remember.

Reply:
Nope, you're right. It was a peer of `notistack` but got changed.

Context: https://github.com/mui-org/material-ui/pull/14572#discussion_r257794673